fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/PeoplePickerTokens.kt

```diff
@@ -15,102 +15,16 @@ import com.microsoft.fluentui.theme.token.FluentGlobalTokens
-    }
+open class PeoplePickerTokens : TextFieldTokens() {
 
     @Composable
     open fun chipSpacing(peoplePickerInfo: PeoplePickerInfo): Dp {
         return FluentGlobalTokens.size(FluentGlobalTokens.SizeTokens.Size80)
     }
 
-    @Composable
-    open fun minHeight(peoplePickerInfo: PeoplePickerInfo): Dp {
-        return FluentGlobalTokens.size(FluentGlobalTokens.SizeTokens.Size480)
-    }
-
-    @Composable
-    open fun textColor(peoplePickerInfo: PeoplePickerInfo): Color {
-        return FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground1].value(
-            themeMode = FluentTheme.themeMode
-        )
-    }
-
-    @Composable
-    open fun textFieldTypography(peoplePickerInfo: PeoplePickerInfo): TextStyle {
-        return FluentTheme.aliasTokens.typography[FluentAliasTokens.TypographyTokens.Body1]
-    }
-
-    @Composable
-    open fun cursorBrush(peoplePickerInfo: PeoplePickerInfo): Brush {
-        return SolidColor(
-            FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
-                themeMode = FluentTheme.themeMode
-            )
-        )
-    }
-
-    @Composable
-    open fun labelTypography(peoplePickerInfo: PeoplePickerInfo): TextStyle {
-        return FluentTheme.aliasTokens.typography[FluentAliasTokens.TypographyTokens.Caption2]
-    }
-
-    @Composable
-    open fun labelColor(peoplePickerInfo: PeoplePickerInfo): Color {
-        return if (peoplePickerInfo.isStatusError)
-            FluentTheme.aliasTokens.errorAndStatusColor[FluentAliasTokens.ErrorAndStatusColorTokens.DangerForeground1].value()
-        else if (peoplePickerInfo.isFocused)
-            FluentTheme.aliasTokens.brandForegroundColor[FluentAliasTokens.BrandForegroundColorTokens.BrandForeground1].value()
-        else
-            FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground2].value()
-
-    }
-
-    @Composable
-    open fun assistiveTextColor(peoplePickerInfo: PeoplePickerInfo): Color {
-        return if (peoplePickerInfo.isStatusError)
-            FluentTheme.aliasTokens.errorAndStatusColor[FluentAliasTokens.ErrorAndStatusColorTokens.DangerForeground1].value()
-        else
-            FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground2].value()
-
-    }
-
-    @Composable
-    open fun assistiveTextTypography(peoplePickerInfo: PeoplePickerInfo): TextStyle {
-        return FluentTheme.aliasTokens.typography[FluentAliasTokens.TypographyTokens.Caption2]
-    }
-
-    @Composable
-    open fun assistiveTextPadding(peoplePickerInfo: PeoplePickerInfo): PaddingValues {
-        return PaddingValues(
-            top = FluentGlobalTokens.size(FluentGlobalTokens.SizeTokens.Size40),
-            bottom = FluentGlobalTokens.size(FluentGlobalTokens.SizeTokens.Size40)
-        )
-    }
-
     @Composable
```

Comment:
hintColor & hintTextTypography are also available in TexFieldTokens. These should come for TexFieldTokens unless required overriding.